data/assets/scene/digitaluniverse/abell.asset

```diff
@@ -34,16 +34,20 @@ local Object = {
       Unit = "Mpc",
       TransformationMatrix = TransformMatrix
     },
-    Color = { 1.0, 0.4, 0.2 },
     Opacity = 1.0,
-    --ColorMap = speck .. "abell.cmap",
+    Coloring = {
+      FixedColor = { 1.0, 0.4, 0.2 },
+      --ColorMap = speck .. "abell.cmap", -- TODO: Decide whether to add
```

Comment:
DiscussDicsuss whether this should be added or removed

Reply:
Looked at the file itself now and it's one of the complicated ones that I quite don't understand. So I'll remove it for now either way.  It's better to add a new color map if they want it, down the line

(It's prepared by Brent Tully (U Hawaii), Stuart Levy (NCSA/UIUC), who I have know idea who they are. Do you?)